client/src/components/QandA/QuestionsList/HelpfulTracker.jsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState } from 'react';
+// import { token, url } from '/config.js';
```

Comment:
remove these commented out lines unless you are saving for later

Reply:
Will do. Once done, I'll resolve all the code where these appear.